src/components/AVPlayerRMP/AvSeekBar.js

```diff
@@ -202,20 +200,12 @@ class AvSeekBar extends Component {
 
       if (typeof clientX !== 'undefined') {
         // pause when dragging handles
-        if (this.sliceStartActive === true || this.sliceEndActive === true) {
-          this.props.media.pause();
-        }
 
         const seekPosition = this.getSeekPositionFromClientX(clientX);
         media.seekTo(seekPosition);
         this.setState({ playPoint: seekPosition });
       }
 
-      // only play if media was playing prior to mouseDown
```

Comment:
Don't we need this functionality in slice mode ?

Reply:
it seems like there is no need for this lines